src/test/run-make/incr-prev-body-beyond-eof/a.rs

```diff
@@ -0,0 +1,14 @@
+fn main() {
+    // foo must be used.
+    foo();
+}
+
+fn foo() {
+    // foo's span in a.rs and b.rs must be identical
```

Comment:
Could you mention the fact that the empty lines in this function are necessary? I'm worried that they'll get accidentally removed in the future.

Reply:
Good idea, will do.